example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js

```diff
@@ -126,6 +126,11 @@ describe('The DiscoverGranules workflow', () => {
       }
     ));
 
+    await waitForApiStatus(
```

Comment:
Should we be waiting for both executions?

Reply:
Yes [`ca5727d` (#2334)](https://github.com/nasa/cumulus/pull/2334/commits/ca5727df7dfd1c035245249a804b9eea5187be88)